site/source/docs/index.rst

```diff
@@ -51,7 +51,6 @@ The full hierarchy of articles, opened to the second level, is shown below.
   compiling/index
   building_from_source/index
   contributing/index
-  api_reference/index
```

Comment:
What happened to api_reference?

Reply:
When building the site I get multiple warnings like ```/emscripten/site/source/docs/api_reference/index.rst: document is referenced in multiple toctrees: ['docs/index', 'index'], selecting: index <- docs/api_reference/index```.

I thought about deal with them by removing the duplicates but then I changed my mind to make the pull request easier to understand, but I forgot to add ```api_reference/index``` back.